biscuit-auth/src/lib.rs

```diff
@@ -59,7 +59,7 @@
 //!   // we want to limit access to `/a/file1.txt` and to read operations
 //!   let token2 = {
 //!     // the token is deserialized, the signature is verified
-//!     let deser = Biscuit::from(&token1,  root.public())?;
+//!     let deser = Biscuit::from(&token1, root.public())?;
```

Comment:
it's good to know it works well in both cases, but maybe we should harmonize the examples?

imo passing it by reference should be the default in examples. If we want to make sure it works both ways (and we should!), ensuring it in tests would be good and would not affect documentation.

Reply:
I put both to show all possibilities as you were concerned by API breaking changes. 

But IMO I find reference only examples better!

I'm waiting for @Geal approval to make final modifications.

Reply:
references should work better for the examples yes